sdk/communication/Azure.Communication.Sms/tests.int.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
```

Comment:
Why did you split this out? My understanding is Int testing is no longer supported because we cannot test in a production environment. If so, can you please delete this instead?